File: FantasticTour/Models/Paquete.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FantasticTour.Models
{
    public class Paquete
    {
        public int Id { get; set; }
        public String Nombre { get; set; }
        public DateTime FechaVencimiento { get; set; }
        public bool Disponible { get; set; }
        public Transporte Transporte { get; set; }
        public Estadia Estadia { get; set; }
        public Atraccion Atraccion { get; set; }
    }
}

File: FantasticTour/ClientApp/app/models/paquete.ts
import { Transporte } from './transporte'
import { Estadia } from './estadia'
import { Atraccion } from './atraccion'

export class Paquete {
    id: number;
    nombre: string;
    fechaVencimiento: string;
    disponible: boolean;
    transporteId: number;
    estadiaId: number;
    atraccionId: number;
    atraccion: string;
    hotel: string;
    costo: number;
}